portfolio/src/main/webapp/script.js

```diff
@@ -12,23 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Load the Visualization API and the corechart package.
+google.charts.load('current', {packages:['wordtree']});
+
+// Set a callback to run when the Google Visualization API is loaded.
+google.charts.setOnLoadCallback(drawChart); 
+
 /**
  * Fetches comments and appends them as list elements.
  */
 function getComments() {
   const commentLimit = document.getElementById('limit-comments').value;
+
+  // Get the comments from the database.
   fetch('/data?limit-comments=' + commentLimit)
       .then(response => response.text())
       .then((comments) => {
-    const commentsObj = JSON.parse(comments);
-    const commentGrid = document.getElementById('comments-grid');
-    commentGrid.innerHTML = '';
-    commentsObj.forEach((comment) => {
-      const li = document.createElement('li');
-      li.innerText = comment;
-      li.classList.add('list-group-item');
-      commentGrid.appendChild(li);
-    });
+        console.log(comments)
+        const commentGrid = document.getElementById('comments-grid');
+        const commentsObj = JSON.parse(comments);
+        commentGrid.innerHTML = '';
+        commentsObj.forEach((comment) => {
+          const li = document.createElement('li');
+          li.append(Object.keys(comment)[0], ': ', 
+              Object.values(comment)[0]);
+          li.classList.add('list-group-item');
+          commentGrid.appendChild(li);
+        });
+      });
+}
+
+/**
+ * Determine if user is logged in and display comment submission if logged.
```

Comment:
Begin JsDoc function descriptions with a verb+"s" typically. So in this case, "Determines if ..."